packages/internet-header/src/components/post-internet-breadcrumbs/post-internet-breadcrumbs.tsx

```diff
@@ -295,19 +300,23 @@ export class PostInternetBreadcrumbs {
               clickHandler={() => this.handleToggleDropdown()}
             ></BreadcrumbList>
           </nav>
-          <div class="breadcrumb-buttons">
-            {breadcrumbConfig.buttons.map(button => (
-              <button
-                class="btn btn-secondary btn-icon"
-                key={button.text}
-                aria-expanded={`${this.overlayVisible && this.currentOverlay === button.overlay}`}
-                onClick={() => this.toggleOverlay(button.overlay, true)}
-              >
-                <SvgIcon name={button.svgIcon.name}></SvgIcon>
-                <span class="visually-hidden">{button.text}</span>
-              </button>
-            ))}
-          </div>
+          {!this.hideButtons && (
+            <div class="breadcrumb-buttons">
+              {breadcrumbConfig.buttons.map(button => (
+                <button
+                  class="btn btn-secondary btn-icon"
+                  key={button.text}
+                  aria-expanded={`${Boolean(
```

Comment:
Boolean was added as well to prevent having "undefined" inside aria-expanded attribute.